code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -69,6 +70,12 @@ var/global/list/datum/stack_recipe/metal_recipes = list ( \
 	sheettype = "metal"
 	stack_id = "metal"
 
+	radial_buildables = list(
```

Comment:
you should try add barbed wire at least (/effect that takes you to another radial to pick amount)?

Reply:
yea nah at least not in this PR